external/googleapis/renovate.sh

```diff
@@ -29,6 +29,8 @@ REPO="googleapis/googleapis"
 BRANCH="master"
 COMMIT=$(curl -fsSL -H "Accept: application/vnd.github.VERSION.sha" \
   "https://api.github.com/repos/${REPO}/commits/${BRANCH}")
+PIPERORIGIN_REVID=$(gh api "repos/${REPO}/commits/${COMMIT}" |
```

Comment:
Note that not all commits include a `PiperOrigin-RevId`, for example:

https://github.com/googleapis/googleapis/commit/a6bbdede82cbd03af41effc59373cdcd25bf9ed4

My suggestion in the bug was to list the last $N$ commits and pick the latest PiperOrigin-RevId value.  That could fail, but seems unlikely.

Reply:
I see.  Thanks.  Let me take another look.

Reply:
OK, I added the reverse scan.

At this stage though, I don't grok whether this satisfies the requirements.  So, we can either plow ahead and see, or revert and regroup to try another day.